theano/gpuarray/opt.py

```diff
@@ -215,6 +267,232 @@ def apply(self, fgraph):
                     0, 'fast_run', 'fast_compile', 'merge')
 
 
+class GraphToGPU(NavigatorOptimizer):
+    """
+    Transfer the graph as a whole to GPU instead of transfering node by node.
+
+    Parameters
+    ----------
+    local_optimizers_all : List or Set
```

Comment:
right.

On Fri, Jul 8, 2016 at 3:26 PM, Ramana Subramanyam <notifications@github.com
> wrote:

> In theano/gpuarray/opt.py
> <https://github.com/Theano/Theano/pull/4570#discussion_r70127876>:
>
> > @@ -215,6 +267,232 @@ def apply(self, fgraph):
> >                      0, 'fast_run', 'fast_compile', 'merge')
> >
> >
> > +class GraphToGPU(NavigatorOptimizer):
> > +    """
> > +    Transfer the graph as a whole to GPU instead of transfering node by node.
> > +
> > +    Parameters
> > +    ----------
> > +    local_optimizers_all : List or Set
>
> List or SortedSet right ? I don't think it'd be list(set(local_opt)) ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4570/files/a2129e028c69951bb57140e4120c5e5da427087b#r70127876>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALC-4Bb9_FmVu7ddYT3hOcLDsS9XrRlks5qTqRZgaJpZM4Ir4rI>
> .
>